src/components/swap/TradePrice.tsx

```diff
@@ -27,10 +29,18 @@ const StyledPriceContainer = styled.button`
   user-select: text;
 `
 
+function getOneCurrencyRawAmount(currency: Currency): string {
+  return parseUnits('1', currency.decimals).toString()
+}
```

Comment:
this might be useful to you here: https://github.com/Uniswap/interface/blob/main/src/lib/utils/tryParseCurrencyAmount.ts